api/interservice/compliance/ingest/events/inspec/helper_methods.go

```diff
@@ -18,16 +20,22 @@ const (
 // "waived": "no" / "no_expired" / "yes" / "yes_run"
 
 // Status calculates the overall status of a control based on all results
-func (control *Control) Status() (status string) {
+func (control *Control) Status(waived string) (status string) {
 	status = ResultStatusPassed
 	for _, result := range control.Results {
+		// logrus.Infof("************************** CONTROL STATUS *******************************: %+v", result.Status)
+		// logrus.Infof("************************** CONTROL WaiverData *******************************: %+v", control.WaiverData)
 		if result.Status == ResultStatusFailed {
 			status = ResultStatusFailed
 			break
 		} else if result.Status == ResultStatusSkipped {
 			status = ResultStatusSkipped
 		}
 	}
+	if waived == "yes_run" || waived == "yes" {
```

Comment:
This means it will over-ride every other status, do we need to talk to Ankur about this?